infrastructure/disk.tf

```diff
@@ -114,9 +114,51 @@ resource "aws_s3_bucket" "data_refinery_compendia_bucket" {
   }
 }
 
-resource "aws_s3_bucket_metric" "compendia_metrics" {
+resource "aws_s3_bucket_metric" "compendia_bucket_metrics" {
   bucket = "${aws_s3_bucket.data_refinery_compendia_bucket.bucket}"
-  name   = "data-refinery-compendia-metric-${var.user}-${var.stage}"
+  name   = "EntireBucket"
+}
+
+resource "aws_cloudwatch_event_rule" "compendia_object_metrics" {
+  name = "data-refinery-compendia-object-metric-${var.user}-${var.stage}"
+  description = "Download Compendia Events"
+
+  event_pattern = <<PATTERN
+{
+  "source": [
+    "aws.s3"
+  ],
+  "detail-type": [
+    "AWS API Call via CloudTrail"
+  ],
+  "detail": {
+    "eventSource": [
+      "s3.amazonaws.com"
+    ],
+    "eventName": [
+      "GetObject"
+    ],
+    "requestParameters": {
+      "bucketName": [
+        "data-refinery-s3-compendia-${var.user}-${var.stage}"
+      ]
+    }
+  }
+}
+PATTERN
+}
+
+# arn needs to be stripped of trailing `:*`
+# aws appends this when creating the event_target
+resource "aws_cloudwatch_event_target" "compendia_object_metrics_target" {
+  rule = "${aws_cloudwatch_event_rule.compendia_object_metrics.name}"
+  target_id = "compendia-object-logs-target-${var.user}-${var.stage}"
+  arn = "${substr(aws_cloudwatch_log_group.compendia_object_metrics_log_group.arn,0,length(aws_cloudwatch_log_group.compendia_object_metrics_log_group.arn) - 2)}"
+}
+
+# Must start with `/aws/events` in order to connect to a cloudwatch_event_target.
+resource "aws_cloudwatch_log_group" "compendia_object_metrics_log_group" {
+  name = "/aws/events/data-refinery-compendia-log-group-${var.user}-${var.stage}"
```

Comment:
I think `logging.tf` might be a slightly more appropriate home for this resource.